index.md

```diff
@@ -19,7 +19,31 @@ layout: default
             <section>
                <p class="component-body">The Fabric Loader download above is the bare minimum. Combine it with Fabric API to get all the important extra APIs mods use. It has to be put into the mods folder like any other mod.</p>
             </section>
-            <a class="button secondary" href="https://www.curseforge.com/minecraft/mc-mods/fabric-api/files">Download Fabric API</a>
+            <div class="download-api-container">
+                <a class="button secondary" href="https://www.curseforge.com/minecraft/mc-mods/fabric-api/files">
+                    <img alt="CurseForge logo" src="/assets/icons/curseforge-logo.svg" width="20" />
+                    Download Fabric API
+                </a>
+                <div class="download-api" tabindex="-1">
+                    <div class="button secondary">
+                        <span>▼</span>
+                    </div>
+                    <div class="download-api-dropdown">
+                        <a href="https://www.curseforge.com/minecraft/mc-mods/fabric-api/files">
+                            <img alt="CurseForge logo" src="/assets/icons/curseforge-logo.svg" width="20" />
```

Comment:
Sorry to be annoying, but where did the .svg images come from? Do we have permission to use them like this? If so please add a text file next to them stating this.

Reply:
So, the Github one comes from the free set of FontAwesome icons and thus has the header license included in it's SVG file

The modrinth and curseforge ones both lack official brand guidelines so they've both been copied over from their respective logos. 

Not sure of any other way of going about it unless we just don't use them? Happy for feedback, not using them isn't the end of the world.